src/index.html

```diff
@@ -1,29 +1,29 @@
 <!DOCTYPE html>
-<html lang='en'>
-<head>
-    <meta charset='UTF-8' />
-    <meta content='width=device-width, initial-scale=1' name='viewport' />
-    <title>Graphhopper Maps</title>
-    <style>
-        html,
-        body {
-            height: 100%;
-            width: 100%;
-            margin: 0;
-            padding: 0;
-            font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
-        }
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta content="width=device-width, initial-scale=1" name="viewport" />
```

Comment:
Interesting. Here it uses " instead of ' ...

Reply:
Yes, and also in JSX: https://github.com/graphhopper/graphhopper-maps/pull/88#discussion_r643045987

Reply:
Aha, ok. Probably because of some xml compatibility. Should we use " instead of ' in ts/x?

Reply:
I don't know. @Janekdererste ?

Reply:
Rather not, but I don't have a strong oppinion on that. 🤷‍♀️

Reply:
So let's keep the double quotes for html and tsx and merge?

Reply:
I'd vote yes. Since Prettier is supposed to be an oppinionated formatter, we should stick to the defaults as long as nobody has a strong oppinion to do things otherwise.

Reply:
:+1: